lib/broadside/ecs/ecs_manager.rb

```diff
@@ -44,24 +44,43 @@ def get_latest_task_definition_arn(name)
         get_task_definition_arns(name).last
       end
 
+      def get_running_instance_ips!(cluster, family, task_arns = nil)
+        ips = get_running_instance_ips(cluster, family, task_arns)
+        raise Error, "No running tasks found for '#{family}' on cluster '#{cluster}'!" if ips.empty?
+        ips
+      end
+
       def get_running_instance_ips(cluster, family, task_arns = nil)
         task_arns = task_arns ? Array.wrap(task_arns) : get_task_arns(cluster, family)
-        raise Error, "No running tasks found for '#{family}' on cluster '#{cluster}'!" if task_arns.empty?
 
-        tasks = ecs.describe_tasks(cluster: cluster, tasks: task_arns).tasks
-        container_instances = ecs.describe_container_instances(
-          cluster: cluster,
-          container_instances: tasks.map(&:container_instance_arn),
-        ).container_instances
+        if task_arns.empty?
```

Comment:
altered so that it does not raise an error. that's what the `!` method will be used for